components/brave_wallet_ui/components/desktop/views/portfolio/components/nft-details/index.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react'
+
+// Types
+import {
+  BraveWallet,
+  NFTMetadataReturnType,
+  DefaultCurrencies
+} from '../../../../../../constants/types'
+
+// Utils
+import { hexToNumber } from '../../../../../../utils/format-balances'
+import { CurrencySymbols } from '../../../../../../utils/currency-symbols'
+import { getLocale } from '../../../../../../../common/locale'
+
+// Styled Components
+import {
+  StyledWrapper,
+  NFTImage,
+  DetailColumn,
+  TokenName,
+  TokenFiatValue,
+  TokenCryptoValue,
+  DetailSectionRow,
+  DetailSectionColumn,
+  DetailSectionTitle,
+  DetailSectionValue,
+  ProjectDetailRow,
+  ProjectDetailName,
+  ProjectDetailDescription,
+  ProjectDetailImage,
+  ProjectDetailButtonRow,
+  ProjectDetailButton,
+  ProjectDetailButtonSeperator,
+  ProjectWebsiteIcon,
+  ProjectTwitterIcon,
+  ProjectFacebookIcon,
+  ProjectDetailIDRow,
+  ExplorerIcon,
+  ExplorerButton
+} from './style'
+
+export interface Props {
+  selectedAsset: BraveWallet.BlockchainToken
+  nftMetadata: NFTMetadataReturnType
+  defaultCurrencies: DefaultCurrencies
+  selectedNetwork: BraveWallet.EthereumChain
+}
+
+const NFTDetails = (props: Props) => {
+  const {
+    selectedAsset,
+    nftMetadata,
+    defaultCurrencies,
+    selectedNetwork
+  } = props
+
+  const onViewOnBlockExplorer = () => {
```

Comment:
Can we use `useExplorer` here? Maybe there's something to change in the hook itself.

Reply:
Updated and is now using the `useExplorer` hook!